Source/WebInspectorUI/UserInterface/Views/ColorSwatchDisplay.css

```diff
@@ -0,0 +1,19 @@
+.swatch-display{
+    margin: 0.5rem 0;
+    font-size: 10px;
+    font-weight: bold;
+    min-height: 0px;
+    max-height: 70px;
+}
+
+.swatch-display .swatch-wrapper{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 3px;
```

Comment:
I recently was thinking if perhaps grid would be a better option compared to flex for the layout for these swatches. Do you think grid would be a better option ?

Reply:
you could experiment and see what it's like with grid, but i think what you have for this is fine since ultimately what we want is "a list of things that wrap" :)